hack/preload-images/generate.go

```diff
@@ -137,6 +137,10 @@ func imagePullCommand(containerRuntime, img string) *exec.Cmd {
 	if containerRuntime == "containerd" {
 		return exec.Command("docker", "exec", profile, "sudo", "crictl", "pull", img)
 	}
+
+	if containerRuntime == "cri-o" {
+		return exec.Command("docker", "exec", profile, "sudo", "crictl", "pull", img)
```

Comment:
this is exactly same as containerd ...could it be an || ?

Reply:
It could be the same for docker as well, I suppose